book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] [REQUIRED] Good job so far! Getters and Setters for `@rentals` should be present in `attr_accessor` of `Book`, please add `:rentals` to that 🙏🏼 

```ruby
attr_accessor :title, :author, :rentals
```